content/en/tracing/tracing_without_limits/_index.md

```diff
@@ -0,0 +1,220 @@
+</div>
+
+### Ingestion Controls
+
+{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-1.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
+
+Ingestion Controls affect what is sent by your applications to Datadog.
+
+#### Span Ingestion
+
+{{< img src="tracing/trace_indexing_and_ingestion/DataIngestion.png" style="width:100%;" alt="Data Ingestion" >}}
+
+Many instrumented services will send 100% of their traffic to Datadog by default.  High-volume services, or services that experience intermittent traffic.
+
+**Note:** If you are seeing numbers below 100% for Ingestion Rate, ensure you are using Agent 6.19+ or 7.19+ as these versions increased the default rate.
+
+In app, on the ['Data Ingestion' view][4], you can see the below information:
+
+| Column                | Data |
+| ----------------------- | ---------- |
+| Service                 | The name of each service instrumented and sending traces to Datadog.   |
+| Data Ingested             | Amount of data ingested by Datadog over the selected time period.      |
+| Ingestion Rate                 | A percentage from 0 to 100% of how many spans produced by the service that are being ingested by Datadog.  Any number lower than 100% means       |
+| Status             | Will show `Default` unless changed by using the instructions in-app to configure the tracer. See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) for more information.    |
+| Requests per second                 |   Average number of requests per second received by the service over the selected time period. Services with intermittent traffic or high volume are likelier to not send 100% of spans by default to Datadog.    |
+| Spans Ingested            | Number of spans ingested by Datadog over the selected time period.        |
+
+#### Change the Default Ingestion Rate
+
+{{< img src="tracing/trace_indexing_and_ingestion/ChangeIngestRate.gif" style="width:100%;" alt="Change the Data Ingestion Rate" >}}
+
+For any services that you want to send more traffic than the default percentage this can be configured by adding a generated code snippet to your tracer configuration for that service to ensure that percentage of traffic gets sent, from 0% to 100%
+
+### Indexing Controls
+
+{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-3.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
+
+Indexing Controls affect what is stored by Datadog for 15 days.
+
+#### Retention Filters
+
+After spans have been ingested by Datadog, they will be retained for 15 days according to the indexing filters that have been set on your account.  By default, the only retention filter enabled per service will be the [Intelligent Sampling Filter](#datadog-intelligent-sampling-filter), which will retain error traces and proportional traces representing latency distributions.
```

Comment:
```suggestion
After spans have been ingested by Datadog, they will be retained for 15 days according to the indexing filters that have been set on your account.  By default, the only retention filter enabled per service will be the [Intelligent Sampling Filter](#datadog-intelligent-sampling-filter), which retains error traces and proportional traces representing latency distributions.
```

Reply:
it's not clear to me what "and proportional traces representing latency distributions" means. Any way to phrase that more clearly?